packages/devextreme/js/ui/number_box/number_box.mask.js

```diff
@@ -420,7 +420,11 @@ const NumberBoxMask = NumberBoxBase.inherit({
         const isValueChanged = parsedValue !== this._parsedValue;
 
         const isDecimalPointRestricted = char === decimalSeparator && maxPrecision === 0;
-        const isUselessCharRestricted = !isTextSelected && !isValueChanged && char !== MINUS && !this._isValueIncomplete(editedText) && this._isStub(char);
+        const isUselessCharRestricted =
+            !isTextSelected
```

Comment:
It's a kind of personal taste but don't u want to move `!isTextSelected` to the previous row? Naughty `=` looks strange to me :)

Reply:
ok, thanks. fixed